addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -144,7 +146,7 @@ def _compute_peppol_endpoint(self):
                 eas_to_field = EAS_MAPPING[partner.country_code]
                 # Try to set both the peppol_eas and the peppol_endpoint
                 for field_name in eas_to_field.values():
-                    if field_name and field_name in partner._fields and partner[field_name]:
+                    if field_name and field_name in partner._fields and partner[field_name] and not partner.peppol_endpoint:
```

Comment:
```suggestion
                    if field_name and field_name in partner._fields and partner[field_name] and partner.peppol_endpoint not in eas_to_field:
```
?
To me, we need to recompute the peppol_endpoint if you changed the country of your partner